src/components/Conversion/ConversionNOB.vue

```diff
@@ -156,23 +163,17 @@ export default class NatureOfBusiness extends Vue {
   }
 
   /** Called when this edit mode has changed. */
-  @Watch('onEditMode')
+  @Watch('onEditMode', { immediate: true })
```

Comment:
Making this an Immediate Watcher sets the initial value of the `isValidNatureOfBusiness` flag.

(It's initialize to True already in the store but this is good practice.)